### ISSUE TITLE
[enhancement] - Better error messages

### ISSUE THREAD

**Issue by Psykar** [gajus/babel-plugin-react-css-modules#57]
https://github.com/gajus/babel-plugin-react-css-modules/blob/f3ccba1d182a465fe81bba9f1217ccc5e0ac08df/src/getClassName.js#L25

Should put the name that fails here, otherwise you have to check each styleName manually.

**Comment by Psykar** [comment]
Maybe not required - these are very early import messages, not after the file is parsed, and in my case were due to screwing up my config ( #58 ) 
Might still be useful - but if you disagree and close, I won't be fussed :)

**Comment by mzvonar** [comment]
Yes, that would be really helpful. It would be much easier to find missing class

**Comment by NE-SmallTown** [comment]
https://github.com/gajus/babel-plugin-react-css-modules/blob/f3ccba1d182a465fe81bba9f1217ccc5e0ac08df/src/index.js#L109-L111

here we can add
```
if (stats.opts.debug) {
  console.log(`at ${filename} import:`);
  console.log(targetResourcePath);
  console.log(`the resource link we use is: ${path.node.source.value}`);
}
```

https://github.com/gajus/babel-plugin-react-css-modules/blob/f3ccba1d182a465fe81bba9f1217ccc5e0ac08df/src/index.js#L134-L136

and here we can add:
```
if (stats.opts.debug) {
  console.log(`after process the import the filenameMap[${filename}].styleModuleImportMap is:  ` + JSON.stringify(filenameMap[filename].styleModuleImportMap) + '\n');
}
```
